Documentation/ExtensionArchitecture/Tutorials/Kickstart/Make/BackendController.rst

```diff
@@ -0,0 +1,56 @@
+.. include:: /Includes.rst.txt
+.. index::
+   Extension development; Make backend controller
+.. _extension-make-backend-controller:
+
+===============================
+Create a new backend controller
+===============================
+
+If you do not have one yet, :ref:`create a basic extension <extension-make>`
+to put the controller in.
+
+..  tabs::
+
+    ..  group-tab:: Composer
+
+        ..  code-block:: bash
+
+            vendor/bin/typo3 make:backendcontroller
+
+    ..  group-tab:: DDEV, Composer
+
+        ..  code-block:: bash
+
+            ddev exec vendor/bin/typo3 make:backendcontroller
```

Comment:
```suggestion
            ddev typo3 make:backendcontroller
```

Ad we had this discussion before, I opt for removing it from here as we don't add it for other cases (e.g. commands).

Reply:
Ok, removed it